packages/block-library/src/list-item/hooks/use-split.js

```diff
@@ -1,23 +1,32 @@
 /**
  * WordPress dependencies
  */
-import { useCallback } from '@wordpress/element';
+import { useCallback, useRef } from '@wordpress/element';
 import { useSelect } from '@wordpress/data';
 import { store as blockEditorStore } from '@wordpress/block-editor';
-import { cloneBlock } from '@wordpress/blocks';
+import { cloneBlock, createBlock } from '@wordpress/blocks';
 
 export default function useSplit( clientId ) {
+	// We can not rely on the isAfterOriginal parameter of the callback,
```

Comment:
There is an issue with the GitHub diff. This code is in no way related to this PR and was already merged. I guess we need a rebase?